DMCompiler/Compiler/DM/AST/DMAST.cs

```diff
@@ -4,6 +4,21 @@ namespace DMCompiler.Compiler.DM.AST;
 
 public abstract class DMASTNode(Location location) {
     public readonly Location Location = location;
+
+    public override string ToString() {
+        return $"{ToString(null)}";
+    }
+
+    public string ToString(Location? loc, bool hideLoc = false) {
```

Comment:
I don't see `hideLoc` used anywhere, and it seems redundant when you can give `null` for the first argument

Reply:
Ah forgot about null

And I've tried to uh, make the thing useful for debugging and stuff. Used it with the param when dealing with tokens and shit.